R/add_CI.R

```diff
@@ -37,31 +37,53 @@ add_CI <- function(gg, ...){
 #' @method add_CI ggsurvfit
 #' @export
 
-add_CI.ggsurvfit <- function(gg, alpha = 0.1, style = "ribbon", linetype = 2, ...){
-
+add_CI.ggsurvfit <- function(gg, 
+                             alpha = 0.1, 
+                             style = "ribbon", 
+                             linetype = "2a", ...){
```

Comment:
Linetype "2a" is not a sensible default,, neither is a hack.  Please update. I dont understand the issue. Perhaps we can discuss this to look for a solution? I never had an issue with this function honestly.